pravega-cli/README.md

```diff
@@ -0,0 +1,117 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega CLI
+
+Pravega CLI is a tool for inspecting and exploring Pravega deployments.
+
+## Prerequisites
+
+- Java 8+
+
+## Building Pravega CLI
+
+Checkout the source code:
+
+```
+git clone https://github.com/pravega/pravega-tools
+cd pravega-tools
+```
+
+Build the Pravega CLI tool:
+
+```
+./gradlew distTar
+```
+
+Unzip the result distribution artifact:
+
+```
+cd pravega-cli\build\distributions\
```

Comment:
This is a Windows path, replace backward slashes with forward slashes.

Reply:
The same applies to all other occurrences.

Reply:
Fixed.